mlir_graphblas/src/lib/GraphBLAS/GraphBLASLowerPass.cpp

```diff
@@ -1119,6 +1119,10 @@ class LowerApplyRewrite : public OpRewritePattern<graphblas::ApplyOp> {
                 return quotient;
               });
 
+    } else if (apply_operator == "second") {
+
+      transformResult = thunk;
```

Comment:
Does this get the correct results with something like the following?
```
%matrix_answer = graphblas.apply %thunk, %sparse_matrix { apply_operator = "second" } : (i64, tensor<?x?xi64, #CSR64>) to tensor<?x?xi64, #CSR64>
```
I think the above syntax is valid. This PR's behavior is unintuitive given this code.

I think that this might be the implementation we want since it'll serve all of our needs, but it might be nicer to name it something like "replace" or "fill". I'm leaning towards the latter because of [`np.fill`](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.fill.html). This sort of name will make it more clear that the ordering doesn't matter.

What are your thoughts?

Reply:
I think we should also open an issue to make sure that all the operators for `graphblas.apply` are documented (in the docs and the TableGen files) if we're going to punt on them for now.

Reply:
I like `fill`. I'll change the name.